roles/common/tasks/ssl.yml

```diff
@@ -7,6 +7,14 @@
             dest=/usr/local/share/ca-certificates/{{ endpoints.main }}.crt
   notify: refresh CAs
 
+- name: Install any additional CA certificates
+  copy:
+    content: "{{ item.content | default(omit) }}"
+    dest: "/usr/local/share/ca-certificates/{{ item.name }}.crt"
+    src: "{{ item.src | default(omit) }}"
+  with_items: "{{ cacerts | default([]) }}"
```

Comment:
This should probably go into the `ssl` hash, referenced as `ssl.extracacerts`, or something like that.